scripts/travis/codegen_verification.sh

```diff
@@ -72,5 +72,10 @@ else
    echo All tidy
 fi
 
+echo "Run shellcheck in e2e_subs"
+pushd test/scripts/e2e_subs
+shellcheck -e SC2154,SC2034,SC2046,SC2053,SC2207,SC2145 -P . -S warning *.sh
```

Comment:
Maybe to try a full path?
```suggestion
shellcheck -e SC2154,SC2034,SC2046,SC2053,SC2207,SC2145 -P "$(dirname \"$0\")/test/scripts/e2e_subs" -S warning *.sh
```

Also, there are also subfolders with tests in `e2e_subs/`